dotnet/src/InternalUtilities/test/MoqExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+#pragma warning disable CS8620 // Argument cannot be used for parameter due to differences in the nullability of reference types.
+
+internal static class MoqExtensions
```

Comment:
Wouldn't this extension be applied to all projects, including those that are not related to testing?

Reply:
Clarified offline and the test extensions are not going to be pulled to non test projects.